<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devicelistform</class>
 <widget class="QWidget" name="devicelistform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="CompleterLineEdit" name="DevAddress"/>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAdd">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../IDE.qrc">
         <normaloff>:/Menu/rc/Menu/plus.png</normaloff>:/Menu/rc/Menu/plus.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.511, y1:0, x2:0.511364, y2:1, stop:0 rgba(255, 255, 255, 245), stop:0.471591 rgba(235, 238, 241, 247), stop:1 rgba(255, 255, 255, 248));</string>
     </property>
     <column>
      <property name="text">
       <string notr="true">Device</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>CMD：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CommandSendLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnHideCmdRecv">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../IDE.qrc">
         <normaloff>:/Menu/rc/Menu/message.png</normaloff>:/Menu/rc/Menu/message.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="CommandRecvTextEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="StautsLabel">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>当前在线人数：?人</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ParticipantStatus">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>192.168.0.0离开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompleterLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">completertool.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../IDE.qrc"/>
 </resources>
 <connections/>
</ui>
